### PR TITLE
Document GPU particles visibility rect/aabb behavior

### LINKED ISSUE #21887: Large Particles2D shapes not rendering unless their center is visible.
**Godot version:**

3.1 Alpha and Master build as of 9/8/2018

**OS/device including version:**

GTX 970 official drivers

**Issue description:**

Large Particles2D shapes such as a box 1500x500 do not render until the center point is visible. Which causes them to just suddenly pop in to existence.

**Steps to reproduce:**

**Minimal reproduction project:**

[Particles2D POPIN.zip](https://github.com/godotengine/godot/files/2363978/Particles2D.POPIN.zip)

### PR DESCRIPTION
**Pull request by Xrayez**
Closes #21887.

### CODE REVIEW

File: doc/classes/GPUParticles2D.xml [hunk 1]
```diff
@@ -71,7 +71,8 @@
 			Particle texture. If [code]null[/code], particles will be squares.
 		</member>
 		<member name="visibility_rect" type="Rect2" setter="set_visibility_rect" getter="get_visibility_rect" default="Rect2( -100, -100, 200, 200 )">
-			Editor visibility helper.
+			The [Rect2] that determines the area of the world part of which needs to be visible on screen for the particle system to be active.
```
Comment (akien-mga) [inline doc/classes/GPUParticles2D.xml:74]: "the area of the world part of which" sounds weird.

Should it just be "the area of the world which"? (same for the original string copied from the 3D one)

Comment (Xrayez) [reply doc/classes/GPUParticles2D.xml:74]: Yes, I thought the same while copying the 3D one.

Could also be:

```suggestion
			The [Rect2] that determines the node's area which needs to be visible on screen for the particle system to be active.
```

Comment (akien-mga) [reply doc/classes/GPUParticles2D.xml:74]: Sounds good yeah, "world" can also be confused with World2D/World3D.

**Review by clayjohn — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**